test/__init__.py

```diff
@@ -795,11 +796,12 @@ def is_topology_type(self, topologies):
             return True
         return False
 
-    def require_cluster_type(self, topologies=[]):  # noqa
+    def require_cluster_type(self, topologies=None):
```

Comment:
Found a bug!

Reply:
Was there a bug here? I don't see it. `topologies` is read only IIUC so the mutable default arg doesn't have and effect.

Reply:
Yeah fair, but nice to see that it catches this case.